react-app/src/redux/game/reducer.js

```diff
@@ -0,0 +1,39 @@
+// import merge from 'lodash.merge';
+
+import { actionNames } from './actions';
+
+const initialState = {
+  history: [{ squares: Array(9).fill(null) }],
+  stepNumber: 0,
+  xIsNext: true,
+  status: '',
+  winner: []
```

Comment:
Be aware that the calculateWinner function returns null when there is no winner. I know that both are falsy values, but you should keep consistency in the no-winner-value